src/app/globals.css

```
@@ -181,6 +183,43 @@
 	contain-intrinsic-size: 0 100px;
 }
 
+/* Prevent message content from overflowing horizontally */
+.streamdown-content {
+	overflow-wrap: anywhere;
+	word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)